### DISCUSSION TITLE
r19base:/20.deferred

### DISCUSSION THREAD

**Discussion by giscus** [yangbo5207/react19feature#25] [General]
# tutorial/use/deferred

http://localhost:5173/tutorial/use/deferred

**Comment by yangbo5207** [comment]
欢迎各位道友添加我的微信好友 icanmeetu 交流学习心得

**Comment by dlgchg** [comment]
我来了

**Comment by xiaoqufengdi** [comment]
这里的List组件应该要用memo包起来吧！

**Reply by yangbo5207** [reply]
需要，如果使用了 Compiler 就不需要了
